ChatExample/ChatExample/.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.{swift}]
```

Comment:
This is a small change, Xcode will pick this up automatically and simply means users don't need to switch between using 2 and 4 indent size, should hopefully lead to less formatting issues